### PR TITLE
Deselection

### PR DESCRIPTION
**Pull request by dandc87**
Depends on #53 

This allows for clearing the current selection

Please review @Jogan or @ireneduke

### CODE REVIEW

**Comment by Abhiseshan** [comment]
Can this please be merged @Jogan @ireneduke

File: library/src/main/java/com/prolificinteractive/materialcalendarview/CalendarDay.java [hunk 1]
```diff
@@ -12,20 +13,78 @@
...
+     * @return CalendarDay set to the specified date
+     */
+    public static @NonNull CalendarDay from(int year, int month, int day) {
+        return new CalendarDay(year, month, day);
+    }
+
+    /**
+     * @return CalendarDay set to the specified date
+     */
+    public static CalendarDay from(Calendar calendar) {
```
Comment (chahinem) [inline library/src/main/java/com/prolificinteractive/materialcalendarview/CalendarDay.java]: Should we add the `Nullable` annotations also ?

Comment (dandc87) [inline library/src/main/java/com/prolificinteractive/materialcalendarview/CalendarDay.java]: Good idea. Can you comment that in #53 too?

File: library/src/main/java/com/prolificinteractive/materialcalendarview/CalendarDay.java [hunk 2]
```diff
@@ -12,20 +13,78 @@
...
+                CalendarUtils.getYear(calendar),
+                CalendarUtils.getMonth(calendar),
+                CalendarUtils.getDay(calendar)
+        );
+    }
+
+    /**
+     * @return CalendarDay set to the specified date
+     */
+    public static CalendarDay from(Date date) {
```
Comment (chahinem) [inline library/src/main/java/com/prolificinteractive/materialcalendarview/CalendarDay.java]: same here

File: library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java [hunk 3]
```diff
@@ -461,7 +471,7 @@ public CalendarDay getMinimumDate() {
      * @param calendar set the minimum selectable date, null for no minimum
      */
     public void setMinimumDate(Calendar calendar) {
```
Comment (chahinem) [inline library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java]: `Nullable` annotation

Comment (dandc87) [inline library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java]: @chahinem done

**Pull request merged**